sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -408,6 +408,14 @@ export class InstitutionService extends RecordDataModelService<Institution> {
     });
   }
 
+  async doesExist(guid: string): Promise<boolean> {
```

Comment:
@ann-aot, @guru-aot, and @dheepak-aot, I will not block this PR due to this, but please in the future consider not using the count to check if there is a row present in the DB. It will not have a major impact because the table will be small, but I will never consider this as the right approach if we need to do something in tables like sims.applications.
This is the below Postgres analysis when using count:

_select count(1) from sims.institutions where guid = 'ca813fd2-f9b7-4762-83f5-9eb1d244d022'_
![image](https://user-images.githubusercontent.com/61259237/132415722-0941d1ce-c939-4fca-8885-84974ab27450.png)

![image](https://user-images.githubusercontent.com/61259237/132415508-2b990215-d9b6-478d-9054-0c3d411f4b5b.png)

This is the Postgres analysis when using a simple where:

_select 1 from sims.institutions where guid = 'ca813fd2-f9b7-4762-83f5-9eb1d244d022'_

![image](https://user-images.githubusercontent.com/61259237/132415608-f0cddcb9-345d-4030-bdb4-fc3e46391bf7.png)

![image](https://user-images.githubusercontent.com/61259237/132415624-ecdd565a-37b4-45aa-9896-770266a49ae0.png)